tester_app/client/bootstrap-sass.config.js

```diff
@@ -0,0 +1,88 @@
+// IMPORTANT: Make sure to keep the customizations defined in this file
+//            in-sync with the ones defined in app/assets/stylesheets/_bootstrap-custom.scss.
+
+// For a reference on customizations,refer to:
+//  https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss
+
+module.exports = {
+  reactOnRailsSassHelper: './assets/stylesheets/_react-on-rails-sass-helper.scss',
```

Comment:
@justin808 I've tested all 8 versions of the app, there are no errors. This works because I added this to the `app/assets/stylesheets/application.css.scss` manifest before even `_pre-bootstrap`. As you said, and I agreed, it doesn't make sense to have the react_on_rails_sass_helper code inside of the bootstrap-specific files because they aren't related to each other. That is the entire reason we have separate files in the first place. Placing it in the `application.css.scss` explicitly also makes it easier to see what's going on for new users.

<img width="1280" alt="screen shot 2015-11-03 at 10 03 47 pm" src="https://cloud.githubusercontent.com/assets/6173488/10928492/cf88e5aa-8276-11e5-95b6-8cd0b317ec70.png">
<img width="1280" alt="screen shot 2015-11-03 at 10 03 52 pm" src="https://cloud.githubusercontent.com/assets/6173488/10928493/cf890652-8276-11e5-9bed-2f4cffec39d7.png">